frontend/test/__support__/ui.tsx

```diff
@@ -88,7 +88,7 @@ export function renderWithProviders(
   if (mode === "public") {
     reducers = publicReducers;
   } else {
-    reducers = mainReducers;
+    reducers = makeMainReducers();
```

Comment:
Do we want to do this? Or use the exported `mainReducers` from `reducers-main.ts`?

Reply:
Yeah, we need to do it here because otherwise this line https://github.com/metabase/metabase/blob/metabot-v3-command-palette/frontend/src/metabase/palette/components/test/PaletteResults/setup.tsx#L143 is called after the `mainReducers` are computed which means that stub reducers are used instead of the real ones. I suppose that when building this is not an issue, because there are different entry points for EE.